.github/workflows/docs/build-docs

```diff
@@ -96,4 +111,4 @@ if __name__ == "__main__":
             MODULES_DIR / "docs" / "build",
             dest,
             dirs_exist_ok=True,
-            )
```

Comment:
Why are you changing this line?

Reply:
that one was PyCharm 👀
I can change it back if you like?